content/articles/how-to-collect-update-and-organize-python-logs/index.md

```diff
@@ -0,0 +1,337 @@
+- [To work with investigating, design your logs in JSON and merge them](#to-work-with-investigating-design-your-logs-in-json-and-merge-them)
+- [Conclusion](#conclusion)
+- [Further activity reading](#further-activity-reading)
+  
+### Modifying your logs' need level and objective
+The `basicConfig` technique for the logging module is the quickest strategy for planning your loggers' ideal direction. However, the [Python documentation](https://docs.python.org/3.7/library/logging.html#logger-objects) proposes making a logger for every module in your application and arranging a logger for each module utilizing `basicConfig` alone can be annoying.
+
+The three significant measures of `basicConfig` are:
+- **level**: plunging succession of events. The available log ranks are `DEBUG`, `INFO`, `WARNING`, `ERROR`, and `CRITICAL`. The default level is `WARNING`.
+- **controller**: determines where the application will pipe your logs.
+- **design**: messages are logged in the format: `LEVEL>:LOGGER NAME>: MESSAGE>`.
+  
+You could be losing out on truly low logs that can help with underlying cause evaluation, based on how the logging module provides `WARNING` and, more important, level logs as a replacement. 
+
+So instead, use a 'FileHandler' to log single or additional documents on the circle, maybe using a `StreamHandler` or a `SocketHandler` to route logs to the control center or an unessential help over the organization.
+
+#### Benefits of logging documents
+While streaming logs to an outside objective, your application doesn't need to reckon with the possibility of organization-related irregularities. 
+
+Furthermore, in the event of any issues happening while at the same time streaming logs over the organization, you'll not lose admittance to those logs since we'll have upheld them on every server.
+
+It considers making a more customized logging setup, whereby you're ready to isolate files and concentrate those documents with a log following asset.
+  
+#### Representation of basicConfig
+Logs follow a specific design that includes the accompanying attributes:
+- `%(asctime)s`: it yields the date and season of the log, in [local time](https://docs.python.org/3.7/library/time.html#time.asctime).
+- `%(rankname)s` : the logging rank of the information.
+- `%(purport)s`: tenor of the log. Read more about log attributes [here](https://docs.python.org/3/library/logging.html#logrecord-ascribes).
+
+### Make a redid arrangement with different loggers and objections.
+The more your application scales, the more you need to utilize a reliable and explained way of designing each logger by incorporating the log name as a log section. 
+
+Arrange various logs and record their personalities powerfully to self-assertively set the log title to suit the personality of your module. Utilize the logging library's incorporated [getLogger() approach](https://docs.python.org/3.7/library/logging.html#logging.getLogger):

```

Comment:
Do logs have `personalities`? Or did you mean `attributes / properties`?

Reply:
I meant properties